packages/crypto/src/sha.ts

```diff
@@ -1,5 +1,4 @@
-import { sha256 as nobleSha256 } from "@noble/hashes/sha256";
-import { sha512 as nobleSha512 } from "@noble/hashes/sha512";
+import { sha256 as nobleSha256, sha512 as nobleSha512 } from "@noble/hashes/sha2.js";
```

Comment:
Those changes seem to be causing issues for bundlers as in @noble/hashes version 1.x we must not use the file extension. See

https://github.com/paulmillr/noble-hashes/blob/1.8.0/package.json#L126

A user is getting

> Error
  [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './sha2.js' is not defined
  by "exports" in …

I will send a PR to fix this in a bit